app/controllers/mixins/actions/vm_actions/reconfigure.rb

```diff
@@ -233,7 +233,7 @@ def get_vlan_options(host_id)
         def get_iso_options(vm)
           iso_options = []
 
-          datastore_ids = vm.host.storages.pluck(:id)
+          datastore_ids = vm&.host&.storages&.pluck(:id)
```

Comment:
Can't we just return `iso_options` in case the vm has no host? We don't really need to do the DB query here, do we?